rdkit/Chem/Fingerprints/UnitTestSimScreener.py

```diff
@@ -17,21 +17,58 @@ def fingerprinter(mol):
 
 class TestCase(unittest.TestCase):
 
+  def test_SimilarityScreener(self):
+    mol = Chem.MolFromSmiles('C1OCCCC1')
+    probe = fingerprinter(mol)
+
+    screener = SimilarityScreener.SimilarityScreener()
+    self.assertEqual(screener.probe, None)
+    screener.SetProbe(probe)
+    self.assertEqual(screener.probe, probe)
+
+    screener = SimilarityScreener.SimilarityScreener(probe=probe, fingerprinter=fingerprinter)
+    self.assertEqual(screener.probe, probe)
+
+    mol = Chem.MolFromSmiles('CCCN')
+    self.assertEqual(fingerprinter(mol), screener.GetSingleFingerprint(mol))
+
+    screener.Reset()
+
   def test1_TopNScreener(self):
     smis = ['C1CCCCC1', 'C1OCCCC1', 'C1NCCCC1', 'c1ccccc1', 'C1C(C)CCCC1', 'C1C(C)C(C)CCC1']
     suppl = Chem.SmilesMolSupplierFromText('\n'.join(smis), delimiter=",", smilesColumn=0,
                                            nameColumn=-1, titleLine=0)
     metric = DataStructs.TanimotoSimilarity
-    probe = fingerprinter(Chem.MolFromSmiles('C1OCCCC1'))
+    mol = Chem.MolFromSmiles('C1OCCCC1')
```

Comment:
Any particular reason for this change or is it just something you prefer/find more readable?